### PR TITLE
Update 1.0.0-upgrade-guide.html.markdown

### PR DESCRIPTION
**Pull request by themantissa**
## Description

Add an intro paragraph to the upgrade guide.

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ X ] Documentation fix/enhancement

## Required Checklist:

- [X ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X ] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/guides/1.0.0-upgrade-guide.html.markdown [hunk 1]
```diff
@@ -8,6 +8,24 @@ MongoDB Atlas Provider 0.1.0: Upgrade and Information Guide
...
+New features include:
+* Support for Atlas Search
+* Support for Online Archive
+* Support for Data Lake
+* Support for Triggers
+* Re-Architected and Updated Cloud Backup Policy Schedule
+
+1.0.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
+
+In addition, there are a few changes that will require modifications to your TF configuration between earlier versions of the provdier and this one.  Below we have outlined guidance for each of the changes.
```
Comment (coderGo93) [inline website/docs/guides/1.0.0-upgrade-guide.html.markdown]: a typo for `provider` instead of `provdier`

**Review by coderGo93 — CHANGES_REQUESTED** [review]

**Review by threebee — APPROVED** [review]
LGTM, apart from already mentioned typo.

File: website/docs/guides/1.0.0-upgrade-guide.html.markdown (hunk 1 above)
Comment (themantissa) [reply website/docs/guides/1.0.0-upgrade-guide.html.markdown]: @coderGo93 fixed, will request re-review.

**Review by coderGo93 — APPROVED** [review]
LGTM!

**Pull request merged**